tests/footnotes.rs

```diff
@@ -3,40 +3,40 @@
 
 extern crate pulldown_cmark;
 
-include!("normalize_html.rs.inc");
-
 
     #[test]
-    fn spec_test_1() {
+    fn footnotes_test_1() {
         let original = r##"Lorem ipsum.[^a]
 
-[^a]: Cool."##;
+[^a]: Cool.
+"##;
         let expected = r##"<p>Lorem ipsum.<sup class="footnote-reference"><a href="#a">1</a></sup></p>
 <div class="footnote-definition" id="a"><sup class="footnote-definition-label">1</sup>
 <p>Cool.</p>
 </div>
 "##;
 
-        use pulldown_cmark::{Parser, html, Options, OPTION_ENABLE_TABLES, OPTION_ENABLE_FOOTNOTES};
+        use pulldown_cmark::{Parser, html, Options};
 
         let mut s = String::new();
 
         let mut opts = Options::empty();
-        opts.insert(OPTION_ENABLE_TABLES);
-        opts.insert(OPTION_ENABLE_FOOTNOTES);
+        opts.insert(Options::ENABLE_TABLES);
+        opts.insert(Options::ENABLE_FOOTNOTES);
 
         let p = Parser::new_ext(&original, opts);
         html::push_html(&mut s, p);
 
-        assert_eq!(normalize_html(&expected), normalize_html(&s));
+        assert_eq!(expected, s);
     }
```

Comment:
Normalization was added to `new_algo` in #97. Probably just need to re-run the gen-tests.

Reply:
Right, I didn't re-gen, as that was broken, and fixed after this merge commit. I'm wondering what's the best process - get this merge in and fix things up after, or try to get master to a good state and redo this merge? Leaning towards the former because it seems like it blocks things least, but open to other ideas.